x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -42,15 +43,20 @@ export function FieldItem({ field, indexPatternId, highlight }: FieldItemProps)
 
   return (
     <DragDrop
+      value={{ field, indexPatternId: indexPattern.id } as DraggedField}
       dataTestSubj="lnsFieldListPanelField"
-      value={{ field, indexPatternId } as DraggedField}
       draggable
-      className={`lnsFieldListPanel__field lnsFieldListPanel__field-btn-${field.type}`}
+      className={`lnsFieldListPanel__field lnsFieldListPanel__field-btn-${
+        field.type
+      } lnsFieldListPanel__field-${exists ? 'exists' : 'missing'}`}
```

Comment:
nit `exists` and `missing` look like modifiers which are normally separated by double dash: `lnsFieldListPanel__field--${exists ? 'exists' : 'missing'}`